src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Todo } from '../types/Todo';
+import { Filters } from '../types/Filters';
+import cn from 'classnames';
+
+type Props = {
+  currentFilter: Filters;
+  setCurrentFilter: Dispatch<SetStateAction<Filters>>;
+  todos: Todo[];
+  handleClearCompleted: () => Promise<void>;
+};
+
+const filters = [
+  { name: 'All', href: '#/', filter: Filters.All, dataCy: 'FilterLinkAll' },
+  {
+    name: 'Active',
+    href: '#/active',
+    filter: Filters.Active,
+    dataCy: 'FilterLinkActive',
+  },
+  {
+    name: 'Completed',
+    href: '#/completed',
+    filter: Filters.Completed,
+    dataCy: 'FilterLinkCompleted',
+  },
+];
+
+export const TodoFooter: React.FC<Props> = props => {
+  const { todos, handleClearCompleted, currentFilter, setCurrentFilter } =
+    props;
+  const activeTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos.length} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
```

Comment:
let`s remove it in separated component

Reply:
Create separate component for navigation